src/cdk-experimental/popover-edit/edit-ref.ts

```diff
@@ -41,7 +41,8 @@ export class EditRef<FormValue> implements OnDestroy {
 
   constructor(
       @Self() private readonly _form: ControlContainer,
-      private readonly _editEventDispatcher: EditEventDispatcher) {
+      private readonly _editEventDispatcher: EditEventDispatcher,
+      private _ngZone: NgZone) {
```

Comment:
readonly

Reply:
I don't think I'm on board with marking everything as `readonly`. This is a private property which is very unlikely to be changed by something internal. We haven't done it anywhere else in the library and we haven't run into any issues with it so far.

Reply:
It's correct, though; we should be marking `readonly` way more than we do, we just never got in the habit of it

Reply:
Changed, although I feel like it's extremely unlikely that it'll actually catch anything. Furthermore since this is already private, if somebody wanted to override for some reason, there's nothing stopping them from just removing the `readonly`.